cpp/dolfinx/mesh/Mesh.cpp

```diff
@@ -394,6 +395,44 @@ const Geometry& Mesh::geometry() const
   return *_geometry;
 }
 //-----------------------------------------------------------------------------
+bool* Mesh::get_entity_reflections(int cell_n) const
```

Comment:
Is the a property of the `Mesh`, or just a single `Cell`? Or does it belong to `Topology`?

Should it be computed for all cells at once?